lib/src/Texture.cpp

```diff
@@ -1315,23 +1317,48 @@ const void* Texture::data(CubeFace face, unsigned int mipLevel, unsigned int dep
 
 Texture::SaveResult Texture::save(const char* fileName, FileType fileType)
 {
-	if (!converted() || !fileName)
+	if (!fileName)
```

Comment:
Note that this will cause an empty file to be created if this relies on the check in std::ostream overload.

Reply:
good point.. I'll add the converted() check back